### PR TITLE
SBOM hub no longer exists

### PR DESCRIPTION
**Pull request by eccles**
Solution:
Remove all support for the SBOM hub

Fixes [AB#7166](https://dev.azure.com/jitsuin/0629f48c-3979-4bbc-9026-cb06b3dfd0ae/_workitems/edit/7166)

### CODE REVIEW

File: docs/runner/components/events_create.rst [hunk 1]
```diff
@@ -60,32 +60,6 @@ An example when opening a door in Paris:
         confirm: true
 
 
-An example when releasing a software package as an sbom:
```
Comment (j-hartley) [inline docs/runner/components/events_create.rst:63]: we still want this don't we?

File: examples/sbom_release.py [hunk 2]
```diff
@@ -1,118 +0,0 @@
...
-from archivist.constants import ASSET_BEHAVIOURS, SBOM_PACKAGE, SBOM_RELEASE
-from archivist.proof_mechanism import ProofMechanism
-from archivist.utils import get_auth
-
-filterwarnings("ignore", message="Unverified HTTPS request")
-
-ASSET_NAME = "RKVST SaaS Software Package"
-
-
-def sbom_release(arch, release, sbom_filename):
```
Comment (j-hartley) [inline examples/sbom_release.py:23]: we still want this don't we?

File: unittests/testeventsconstants.py [hunk 3]
```diff
@@ -175,59 +156,6 @@
...
-    "sbom": {
-        "filename": "gen1.xml",
-        "content_type": "text/xml",
-        "confirm": True,
-        "params": {
-            "privacy": "PRIVATE",
-        },
-    },
-}
-REQUEST_WITH_SBOM = {
```
Comment (j-hartley) [inline unittests/testeventsconstants.py:195]: we still want these don't we?

File: unittests/testeventscreate.py [hunk 4]
```diff
@@ -23,15 +23,13 @@
     EVENT_ATTRS_ATTACHMENTS,
     EVENT_ATTRS_LOCATION,
     EVENT_ATTRS_LOCATION_IDENTITY,
-    EVENT_ATTRS_SBOM,
```
Comment (j-hartley) [inline unittests/testeventscreate.py:26]: we still want this don't we?

File: docs/runner/components/events_create.rst (hunk 1 above)
Comment (eccles) [reply docs/runner/components/events_create.rst:63]: No - this yaml has an 'sbom:' dict which means that the sbom is created b=y accessing sbom hub whose endpoint has now disappeared.

File: unittests/testeventsconstants.py (hunk 3 above)
Comment (eccles) [reply unittests/testeventsconstants.py:195]: no - this is only used when uploading sbom via sbom hub

File: unittests/testeventscreate.py (hunk 4 above)
Comment (eccles) [reply unittests/testeventscreate.py:26]: no this is only used when testing with uploading using sbom hub

**Review by j-hartley — APPROVED** [review]

**Pull request merged**